icepack/utilities.py

```diff
@@ -46,21 +46,56 @@ def get_kwargs_alt(dictionary, keys, keys_alt):
                   for key, alt_key in zip(keys, keys_alt)))
 
 
-def facet_normal_2(mesh):
+def get_mesh_dimensions(mesh):
```

Comment:
I think all of this would be clearer if, rather than returning a number, you had a helper function that returned a string of the axes involved. So it could return `'x'`, `'xy'`, `'xz'`, or `'xyz'`. When I've referred to things as a 2.5D model, that terminology isn't standard, and it's also kind of debatable anyway. So someone who came across this might go [wat](https://www.destroyallsoftware.com/talks/wat). If you don't want to use strings, you could also define it as an [enum](https://docs.python.org/3/library/enum.html).